internal/tiltfile/flags/flags.go

```diff
@@ -0,0 +1,84 @@
+package flags
+
+import (
+	"github.com/pkg/errors"
+	"go.starlark.net/starlark"
+
+	"github.com/windmilleng/tilt/internal/tiltfile/starkit"
+	"github.com/windmilleng/tilt/internal/tiltfile/value"
+	"github.com/windmilleng/tilt/pkg/model"
+)
+
+type Settings struct {
+	resources []string
+}
+
+type Extension struct {
+}
+
+func NewExtension() Extension {
+	return Extension{}
+}
+
+func (e Extension) NewState() interface{} {
+	return Settings{}
+}
+
+func (Extension) OnStart(env *starkit.Environment) error {
+	return env.AddBuiltin("flags.set_resources", setResources)
+}
+
+func setResources(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
```

Comment:
it was originally something like `enable_resources` and I suggested changing it to `set_resources`, so that it was clear it was overwriting existing state, rather than potentially adding to existing state, at which point maybe it needs to be preceded by a `clear_resources` or something

Reply:
discussed offline, going with `set_enabled_resources`